packages/kit/src/constants.js

```diff
@@ -9,3 +9,6 @@ export const GENERATED_COMMENT = '// this file is generated — do not edit it\n
 export const ENDPOINT_METHODS = ['GET', 'POST', 'PUT', 'PATCH', 'DELETE', 'OPTIONS', 'HEAD'];
 
 export const PAGE_METHODS = ['GET', 'POST', 'HEAD'];
+
+export const SUBSTITUTION_APP_VERSION_HASH = '__SVELTEKIT_APP_VERSION_HASH__';
```

Comment:
These two constants could use a jsdoc comment explaining a bit why we need them (basically a trimmed down version of the PR description)

Reply:
addressed in [`79836ef` (#12700)](https://github.com/sveltejs/kit/pull/12700/commits/79836ef0f453873e4ec37a6728b92b23819cd30f)